bravado/mapping/http_future.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright (c) 2013, Digium, Inc.
+# Copyright (c) 2014, Yelp, Inc.
+#
+
+DEFAULT_TIMEOUT_S = 5.0
+
+
+class HttpFuture(object):
```

Comment:
HttpFuture is now a class instead of an interface. Keeping it in `http_future.py`.